github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/GitHubBranchRepository.java

```diff
@@ -148,6 +150,7 @@ public FormValidation doRunTrigger() throws IOException {
     }
 
     @Override
+    @RequirePOST
```

Comment:
it seems this class and the PR version could be refactored so there is a base class, but that's something for another day.